### PR TITLE
feat: Filtered groups in search view

### PR DESCRIPTION
**Pull request by przemvs**
https://wearezeta.atlassian.net/browse/WPB-10899

## Description

When currently looking for a username under “connect”, the user also gets presented with all groups, that a user is part of that the user is searching for

- This section is called “Team conversations”

- This section “Team conversations” is no longer visible in the “Connect” tab

- This section “Team conversations” should be visible when searching for a username within a filtered view in the conversation list

Filters should be applied, eg. If you are in “Favorites” and you are typing in “John” - only those conversations that have a user “John” in them should be presented to the user in a separate section “Team conversations”

## Checklist

- [x] mentions the JIRA issue in the PR name (Ex. [WPB-423])
- [x] PR has been self reviewed by the author;
- [ ] Hard-to-understand areas of the code have been commented;
- [ ] If it is a core feature, unit tests have been added;

[WPB-423]: https://wearezeta.atlassian.net/browse/WPB-423?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by thisisamir98 — APPROVED** [review]

File: src/script/page/LeftSidebar/panels/Conversations/ConversationsList.tsx [hunk 1]
```diff
@@ -167,18 +180,62 @@ export const ConversationsList = ({
...
+
+    return (
+      <div>
+        <h3 css={headingTitle}>{t('searchGroups')}</h3>
+
+        <ul css={css({margin: 0, paddingLeft: 0})} data-uie-name="conversation-view">
+          {filteredGroupConversations.map((conversation, index) => (
+            <>
+              <ConversationListCell key={conversation.id} {...getCommonConversationCellProps(conversation, index)} />
+            </>
```
Comment (e-maad) [inline src/script/page/LeftSidebar/panels/Conversations/ConversationsList.tsx]: IMO, no need for fragment tag.

File: src/script/page/LeftSidebar/panels/Conversations/ConversationsList.tsx [hunk 2]
```diff
@@ -136,16 +145,20 @@ export const ConversationsList = ({
...
+  const conversationSearchFilter = (conversation: Conversation) => {
+    const filterWord = replaceAccents(conversationsFilter.toLowerCase());
+    const conversationDisplayName = replaceAccents(conversation.display_name().toLowerCase());
 
-        return conversationDisplayName.includes(filterWord);
-      };
+    return conversationDisplayName.includes(filterWord);
+  };
 
+  const getConversationView = () => {
```
Comment (e-maad) [inline src/script/page/LeftSidebar/panels/Conversations/ConversationsList.tsx]: Would be better if we move this to another component.

File: src/script/page/LeftSidebar/panels/Conversations/ConversationsList.tsx [hunk 3]
```diff
@@ -167,18 +180,62 @@ export const ConversationsList = ({
     );
   };
 
+  const getFilteredGroupConversations = () => {
```
Comment (e-maad) [inline src/script/page/LeftSidebar/panels/Conversations/ConversationsList.tsx]: Would be better if we move this to another component.
I think @bardiawire is also suggesting the same.

**Review by e-maad — APPROVED** [review]
LGTM 💯 
Added few minor suggestions.

**Review by e-maad — APPROVED** [review]

**Pull request merged**